core/src/main/scala/cats/SelectiveError.scala

```diff
@@ -0,0 +1,15 @@
+package cats
+
+import cats.syntax.either._
+
+trait SelectiveError[F[_], E] extends ApplicativeError[F, E] with Selective[F] {
```

Comment:
I haven't seen this typeclass in Haskell, but, uh, seems legit?  More MonadError methods could be pushed down.

Reply:
This is out of scope.